Mathlib/Data/Nat/Choose/Vandermonde.lean

```diff
@@ -26,18 +26,17 @@ https://en.wikipedia.org/wiki/Vandermonde%27s_identity#Algebraic_proof .
 
 open BigOperators
 
-open Polynomial Finset.Nat
```

Comment:
I'm a bit confused by this change; isn't `antidiagonal` in the `Finset.Nat` namespace?

Reply:
I did a grep and all other uses showed `Finset Nat`:

```
Mathlib/Combinatorics/SetFamily/LYM.lean:open Finset Nat
Mathlib/Combinatorics/SetFamily/Shadow.lean:open Finset Nat
Mathlib/Combinatorics/SimpleGraph/Regularity/Equitabilise.lean:open Finset Nat
Mathlib/Data/Finset/Slice.lean:open Finset Nat
Mathlib/Data/Nat/Interval.lean:open Finset Nat
Mathlib/Data/Sign.lean:open Finset Nat
Mathlib/Order/Grade.lean:open Finset Nat OrderDual
```

But I shouldn't rely on this result – the formatting of the opens has been restored.